testing/pretty.ts

```diff
@@ -0,0 +1,75 @@
+import { equal } from "./mod.ts";
+import { red, green, white, gray, bold } from "../colors/mod.ts";
+import diff, { DiffType, DiffResult } from "./diff.ts";
+import { format } from "./format.ts";
+
+const CAN_NOT_DISPLAY = "[Cannot display]";
+
+function createStr(v: unknown): string {
+  try {
+    return format(v);
+  } catch (e) {
+    return red(CAN_NOT_DISPLAY);
+  }
+}
+
+function createColor(diffType: DiffType) {
+  switch (diffType) {
+    case "added":
+      return (s: string) => green(bold(s));
+    case "removed":
+      return (s: string) => red(bold(s));
+    default:
+      return white;
+  }
+}
+
+function createSign(diffType: DiffType) {
+  switch (diffType) {
+    case "added":
+      return "+   ";
+    case "removed":
+      return "-   ";
+    default:
+      return "    ";
+  }
+}
+
+function buildMessage(diffResult: ReadonlyArray<DiffResult<string>>) {
+  const messages = [];
+  messages.push("");
+  messages.push("");
+  messages.push(
+    `    ${gray(bold("[Diff]"))} ${green(bold("Added"))} / ${red(
```

Comment:
I wonder if these should be Left and Right (vs Added and Removed) ?

Reply:
@hayd Thanks :) I see. It looks good 👍